icorsi.html

```diff
@@ -9,6 +9,21 @@
   studyYear: '2',   // CCC
 };
 
+const interval = setInterval(() => {
+  const cantoni = ["Modulo per lanciare", "Form to launch"]
```

Comment:
Why are the Swiss cantons mentioned in code?

Reply:
They're 🐂 not 😐 the Swiss [cantons](https://www.youtube.com/watch?v=dQw4w9WgXcQ) but 🧈 the Swiss [Cantoni](https://search.usi.ch/it/persone/c3153bd11a31b6f96024b89b182d629f/cantoni-lorenzo)!